pt_BR/about.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Introdução às dicas de JavaScript
+permalink: pt_BR/about/
+lang: pt_BR
+---
+
+> Ano novo, projeto novo. **Uma dica de JS por dia!**
+
+Com grande entusiasmo, apresento estas dicas diárias de JavaScript curtas e úteis que lhe permitirá melhorar a sua escrita de código. Com menos de 2 minutos a cada dia, você será capaz de ler sobre desempenho, convenções, hacks, questões e todos os itens que o futuro desta linguagem impressionante tem para nós.
```

Comment:
There is no need to translate literally, this can be like:

"... apresento diariamente estas dicas curtas e úteis sobre JavaScript que lhe permitirão melhorar a escrita do **seu** código..."